### DISCUSSION TITLE
Version 0.29 (14 April 2021)

### DISCUSSION THREAD

**Discussion by kinverarity1** [kinverarity1/lasio#442] [General]
- Fix #404 (lasio changes STEP with imprecise floating-point number behaviour; #432)
- Add option ``len_numeric_field=-1``, ``lhs_spacer=" "``, and ``spacer=" "`` to writer.py:write (see #412; PR #418)
- Fix #271 (Read quoted strings in data section properly; #423)
- Fix #427 (Change null_policy to handle small non-zero values; #429)
- Fix #417 (Fix parsing for empty ~Other section; #430)
- Fix #402 (fixes issue when unit starts with dot; #403)
- Fix #395 (Update doc examples to reflect new HeaderItem repr; #410)
- Fix #426 (Update urllib.request to be the preferred urllib; #428)
- Add check for pushed tag version to version tests (#396)
- Update GitHub Action Python CI testing (#399, #400)
- Improve ``las_items.py:HeaderItem.__repr__`` truncation logic (#397)
- Remove codecs import (unused) and fix typo (#406)
- Exclude LAS files from GitHubs Language Stats (#411)
- Re-add try-except check around call to reader.read_data_section_iterative() (#401)
- Remove reader.py:read_file_contents - unused code (see #401; #393)
- Add test for timestring with colon in ~Well section (see #419 - PR #420)
- Fix SyntaxWarning in writer.py (#425)
- Add bugfix and feature request issue templates to GitHub repository
- Apply ``black`` code style to all Python files (#438, #398)
- Update [demo notebook for using logging levels](https://github.com/kinverarity1/lasio/blob/master/notebooks/set%20logging%20level%20for%20lasio.ipynb) with current behaviour
- Update [contributing guide](https://github.com/kinverarity1/lasio/blob/master/docs/source/contributing.rst) (#437, #441)

<hr /><em>This discussion was created from the release <a href='https://github.com/kinverarity1/lasio/releases/tag/v0.29'>Version 0.29 (14 April 2021)</a>.</em>

**Comment by scuervo91** [comment]
Hi. Is there a way to create a LASFile Object from a Json String?

Suppose I have a LASFile Object that I want to save on a DataBase with Json. I would export the LASFile with the method `to_json` and save it, but when querying can I build a LASFile from that json?

Thank you

Santiago Cuervo

**Reply by kinverarity1** [reply]
Hi, sorry I have been slow in replying! It's not currently possible to do that, but it's a great idea. I will post it as an issue so we can hopefully add it in the future.
